### ISSUE TITLE
Board not working

### ISSUE THREAD

**Issue by kwadsten** [rallepalaveev/analog#4]
Hi, I build your v1.4 board but it's not quite working.

The display seems to be working OK in 40 and 80 column modes, but the the disk drive isn't recognized.  It boots right to the Applesoft prompt (with Apple IIE logo at the top of the screen).

When I do a PR#6 to boot the config floppy it doesn't do anything.

I tried adding the jumper, but it didn't help.  I assume the jumper doesn't do anything yet.

Any thoughts?

FYI - I have built the original A2VGA card (https://github.com/Retrotink/Apple-II-VGA) using the same firmware files and that one is working.

**Comment by kwadsten** [comment]
I soldered the NO-PLD jumper on the back and removed the PLD chip and now it's working.  What is the purpose of the PLD?  Does it use the same .jed file as the A2VGA project or do you have a custom one?

**Comment by ThorstenBr** [comment]
The PLD is required for extended functions, e.g. to enable use of the configuration utility to configure the card. Or for the Z80 firmware. Or for A2USB.

This card uses a GAL 22v10. You cannot use the PLD of other projects (e.g. using a GAL 16v8).

There is a PLD file in this project - in the earlier hardware folders. I thought there was also a .jed file somewhere in the project.
Otherwise use this .jed file: https://www.applefritter.com/files/2023/11/17/PICOPAL-N.jed_.zip

**Comment by kwadsten** [comment]
OK, I unsoldered the NO-PLD jumper and re-flashed the PLD with your file.
I do see a .jed in one of your older hardware folders, but I used the one you linked above.
The drive now boots as expected.
  
I also compiled the latest A2VGA pico project .uf2 file ("main" branch) that enables saving.  Now all seems to be working great.  I do have to use a BASIC POKE to save the configuration, but assume they will update their config utility .dsk to support this soon.  (or I can just update my copy of their BASIC configuration program).

Thanks for your response and card design.  It's a beautiful, crisp display in a compact design.
